src/DIRAC/RequestManagementSystem/Agent/RequestExecutingAgent.py

```diff
@@ -75,111 +74,37 @@ class RequestExecutingAgent(AgentModule):
     request processing agent using ProcessPool, Operation handlers and RequestTask
     """
 
-    # # process pool
-    __processPool = None
```

Comment:
why would you remove them from here ? It does not have big interest from the programming POV, but I find it much easier to read

Reply:
to use the same style everywhere

Reply:
Technically this does change these members from class to instance attributes (cf. https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables) 
Except that the `self.__fooBar = 'baz'` assignments create instance attributes replacing (shadowing?) the static members

Reply:
I find it more readable as I made it here, so it is just personal preference.

Unless there is a technical need: why should they stay as class attributes instead of instance attributes?

Reply:
OK but then we may as well do it properly, and add as constant

```
REQUEST_PER_CYCLE = 100
MIN_PROCESS = 20
...
```

Reply:
I have added the constants.